### ISSUE TITLE
The Completed Pomodoro count resets upon losing focus

### ISSUE THREAD

**Issue by a-tooth** [partlyhuman/PblPom#9]
Is this a feature or a bug?  No short-term local history seems to be a drag when anything on the Pebble Time can steal focus.

**Comment by voroninman** [comment]
@partlyhuman Any plans on this?

**Comment by partlyhuman** [comment]
No, you can consider this project abandonware, I haven't had time to look at it for years. There's a remote chance I may upgrade it one day but don't hold your breath. Source is here so anyone is free to fork it and continue.
